src/environment.js

```diff
@@ -0,0 +1,112 @@
+import * as constants from './constants'
+import * as coordinatorApi from './api'
+import { ContractNames } from './constants'
```

Comment:
you're already importing this as constants

Reply:
fixed!